packages/react-client/src/__tests__/ReactFlight-test.js

```diff
@@ -240,7 +244,7 @@ describe('ReactFlight', () => {
       ReactNoop.render(rootModel);
     });
     expect(ReactNoop).toMatchRenderedOutput('Loading...');
-    spyOnDevAndProd(console, 'error');
+    spyOnDevAndProd(console, 'error').mockImplementation(() => {});
```

Comment:
This approach may need further improvement

Reply:
There is a significant difference between Jasmine's spyOn and jest.spyOn, jest.spyOn not only spies on the method but also calls the spied method, whereas Jasmine's spyOn does not call the spied method.

Reply:
Makes sense. So this change is just ensuring equivalence between using jasmine2 and jest-circus, right?

Reply:
Yes, that's correct. However, I feel that this implementation is not elegant.